Assets/MixedRealityToolkit/_Core/Definitions/InputSystem/MixedRealityInputActionsProfile.cs

```diff
@@ -11,6 +12,34 @@ namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.InputSystem
     [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Mixed Reality Input Actions Profile", fileName = "MixedRealityInputActionsProfile", order = 1)]
     public class MixedRealityInputActionsProfile : ScriptableObject
     {
+        private readonly string[] inputActionsMRTKDefaults =
```

Comment:
Not sure if I like the MRTK abbreviation in the name.

`defaultInputActions` would be suitable

Reply:
Renamed